### ISSUE TITLE
Remove browser builds from source

### ISSUE THREAD

**Issue by imor** [qiao/PathFinding.js#60]
Files lib/pathfinding-browser.js and lib/pathfinding-browser.min.js should be removed. These are release artefacts which should be produced only when making a release. For people to have ready access to these files they should be uploaded on the repo's release page.

Benefits of doing this are -
- It will prevent beginners from changing the wrong files. E.g. see https://github.com/qiao/PathFinding.js/pull/32.
- Because the minified file is a single line it is prone to merge conflicts. It will prevent those.

@qiao What do you say?

**Comment by RichardJohnn** [comment]
I had to add this to my bower.json :
```
"overrides": {
    "pathfinding": {
      "main": "pathfinding-browser.js"
    }
  }
```
but even so, there is no PF defined as in  `new PF.Grid(5, 3)`.  

As a comparison, I tried [easystarjs](https://github.com/prettymuchbryce/easystarjs) and also had to add an override to their .js file in bin/ and then ` new EasyStar.js()` just worked

**Comment by imor** [comment]
@RichardJohnn Could you please post your folder structure i.e. where is pathfinding installed and how you import it in your webpage?

**Comment by imor** [comment]
Hi @RichardJohnn , thanks for reporting. Build 0.4.13 was broken as it it did not define the PF variable. I have released 0.4.14. Please try installing again and it should work.

Thanks,
imor

**Comment by RichardJohnn** [comment]
yes, that did the trick!  also, it looks like your bower.json is good to go too, the 'overrides' section in my project's bower.json is not needed anymore
